config/environments/development.rb

```diff
@@ -78,7 +78,9 @@
   config.logout_url = 'http://localhost:9010/logout'
   config.auth_service_url = 'http://localhost:9010'
 
-  config.urls = { submission: '',
+  config.urls = { 
+                  data_release: '',
```

Comment:
These are the URLs for links in the navbar, the API links are kept outside this hash at the moment. I guess we should come up with some standard way for doing this, maybe keep API URLs in a separate hash? I intend to move this hash into the upcoming Rails Engine we'll be using for the shared navbar if that goes ahead.